docs/install/requirements.md

```diff
@@ -61,6 +62,7 @@ Harvester nodes require the following port connections or inbound rules. Typical
 | TCP       | 10250                    | Harvester management and compute nodes  | Kubelet                               |
 | TCP       | 10256                    | Harvester management and compute nodes  | Kube-proxy health checks              |
 | TCP       | 10258                    | Harvester management nodes              | Cloud-controller-manager              |
+| TCP       | 10260                    | Harvester management nodes              | Cloud-controller-manager              |
```

Comment:
```suggestion
| TCP       | 10260                    | Harvester management nodes              | cloud-controller-manager              |
```

Reply:
Yes, if we are using the hyphenated component name as seen in the [Kubernetes documentation](https://kubernetes.io/docs/concepts/architecture/cloud-controller/).